src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -290,9 +290,6 @@ private static void AddLazilyLoadedTraceInstrumentations(LazyInstrumentationLoad
                 case TracerInstrumentation.AspNetCore:
                     DelayedInitialization.Traces.AddAspNetCore(lazyInstrumentationLoader, pluginManager);
                     break;
-                case TracerInstrumentation.MySqlData:
-                    DelayedInitialization.Traces.AddMySqlClient(LazyInstrumentationLoader, pluginManager);
-                    break;
                 case TracerInstrumentation.EntityFrameworkCore:
```

Comment:
```suggestion
                    break;
                case TracerInstrumentation.MySqlData:
                    break;
                case TracerInstrumentation.EntityFrameworkCore:
```

Otherwise it will leads to
```
Logger.Warning($"Configured trace instrumentation type is not supported: {instrumentation}");
                    if (FailFastSettings.Value.FailFast)
                    {
                        throw new NotSupportedException($"Configured trace instrumentation type is not supported: {instrumentation}");
                    }
```